core/log/logger.go

```diff
@@ -0,0 +1,30 @@
+package log
+
+// Logger is the Cosmos SDK logger interface.
+// It maintains as much backward compatibility with the CometBFT logger as possible.
```

Comment:
I would mention that cosmossdk.io/log is the implementation.

Reply:
done